### PR TITLE
Added TouchNet payment variables

### PR DESCRIPTION
**Pull request by bojmil-armedia**

### CODE REVIEW

File: vagrant/provisioning/roles/arkcase-app/templates/server.yml [hunk 1]
```diff
@@ -133,4 +133,10 @@ zylabIntegration:
     clientSecret: "{{ zylab_clientSecret | default('') }}"
     tokenUri: "{{ zylab_tokenUri| default('') }}"
     systemUserEmail: "{{ zylab_systemUserEmail | default('') }}"
-    systemUserPassword: "{{ zylab_systemUserPassword | default('') }}"
\ No newline at end of file
+    systemUserPassword: "{{ zylab_systemUserPassword | default('') }}"
+payment:
```
Comment (david-oc-miller) [inline vagrant/provisioning/roles/arkcase-app/templates/server.yml:137]: do you want to always include this block?  Or include it only if some feature flag is defined, e.g. `touchnet_enabled`?

**Review by dzdimov — APPROVED** [review]

File: vagrant/provisioning/roles/arkcase-app/templates/server.yml (hunk 1 above)
Comment (bojmil-armedia) [reply vagrant/provisioning/roles/arkcase-app/templates/server.yml:137]: I way thinking about it as well. We have these in arkcase.yaml always.
But you might be right. I'll add a flag.

Comment (bojmil-armedia) [reply vagrant/provisioning/roles/arkcase-app/templates/server.yml:137]: Hmm. The same goes probably for zylabIntegration part. Let's merge this one and I will fix both.

Comment (bojmil-armedia) [reply vagrant/provisioning/roles/arkcase-app/templates/server.yml:137]: Created https://arkcase.atlassian.net/browse/DSO-361 to fix this.

**Pull request merged**